ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    training_dataset_bigquery_path = 'bq://bigquery-public-data:iowa_liquor_sales_forecasting.2020_sales_train'
</pre><p> </p><p>Use full-caps for constant variables.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>